tests/Unit/Listeners/SendVoucherHistoryEmailTest.php

```diff
@@ -42,23 +43,23 @@ protected function setUp()
 
         // Set up voucher states.
         Auth::login($this->user);
+
         foreach ($this->vouchers as $v) {
             $v->applyTransition('order');
             $v->applyTransition('print');
             $v->applyTransition('dispatch');
             $v->applyTransition('allocate');
             $v->trader_id = 1;
             $v->applyTransition('collect');
+            $v->applyTransition('confirm');
```

Comment:
What I meant was that there will be vouchers that are not confirmed in the database and the test needs to take into account that it is selecting confirmed ones from a bigger set.

But as I say - this comment is coming out of context so you probably know better.